### PR TITLE
NRPT-497 Create S3 object clean up script

### PR DESCRIPTION
**Pull request by BcGovNeal**
https://bcmines.atlassian.net/browse/NRPT-497

### CODE REVIEW

File: api/src/utils/s3-cleanup.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+    if (result.Contents && result.Contents.length) {
+      contents = contents.concat(result.Contents);
+
+      // Grab the next chunk if there are more results
+      if (result.IsTruncated) {
+        // NRS S3 isn't returning NextMarker, so manually retrieve last key in the array
+        const nextMarker = result.Contents[result.Contents.length - 1].Key;
+
+        const nextContents = await listS3Objects(nextMarker);
```
Comment (KitArmstrong) [inline api/src/utils/s3-cleanup.js:37]: Nice recursion.

File: api/src/utils/s3-cleanup.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  console.log('\nThere are no orphan objects in S3.');
+}
+
+/**
+ * Delete up to 1000 objects from S3.  S3 will only accept up to 1000 keys in a single call
+ *
+ * @param {boolean} objectKeys S3 object keys to delete.
+ * @returns
+ */
+async function deleteS3Objects(objectKeys) {
```
Comment (KitArmstrong) [inline api/src/utils/s3-cleanup.js:127]: Function doesn't need to be async.

**Review by KitArmstrong — APPROVED** [review]
Really nice and clean script. I like the added help text and that you can pass in CLI arguments.

**Review by mtCarto — APPROVED** [review]
Very nice, the CLI help is a good touch

**Pull request merged**